internal/pkg/service/buffer/worker/task/orchestrator/orchestrator_test.go

```diff
@@ -94,19 +94,19 @@ func TestOrchestrator(t *testing.T) {
 [orchestrator][some.task]INFO  ready
 [distribution]INFO  found a new node "node2"
 [orchestrator][some.task]INFO  restart: distribution changed: found a new node "node2"
-[orchestrator][some.task]DEBUG  not assigned "00001000/my-receiver/my-export/ResourceID"
+[orchestrator][some.task]DEBUG  not assigned "00001000/my-receiver/ResourceID"
 %A
 `, d1.DebugLogger().AllMessages())
 
 	wildcards.Assert(t, `
 %A
 [orchestrator][some.task]INFO  ready
-[orchestrator][some.task]INFO  assigned "00001000/my-receiver/my-export/ResourceID"
-[task][%s]INFO  started task "00001000/my-receiver/my-export/some.task/%s"
-[task][%s]DEBUG  lock acquired "runtime/lock/task/00001000/my-receiver/my-export/some.task/ResourceID"
+[orchestrator][some.task]INFO  assigned "00001000/my-receiver/ResourceID"
```

Comment:
Ten task z nějakýho důvodu není assignovanej a netuším proč:
![CleanShot 2023-02-08 at 23 05 08@2x](https://user-images.githubusercontent.com/215660/217661852-613031c9-8a1c-44f1-b962-a0fc7df9b80c.jpg)

Reply:
To znamena, ze neprinalezi danej worker node.

Ak sa menil string podla ktoreho sa to priradzuje (TaskKey? nepamatam si), a  teste su napr. 2 nodes, tak to zobrala proste ta druha, a malo by to tam byt vidiet.

Pozriem na to podrobnejsie neskor (ak by to nepomohlo).

Reply:
(vid `distribution` package)